module-2/lab-matplotlib-seaborn/your-code/challenge-2.ipynb

```diff
@@ -23,18 +23,16 @@
   },
```

Comment:
- When you have temp data is a common practice to plot the series as a lineplot instead of a barplot.  You can also play with xticks in order to have the months in name in the x axis. 
- It should be better to set Steps as your y variable instead of as the x variable.